google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -245,20 +241,7 @@ public void downloadTo(Path path, BlobSourceOption... options) {
    * @param options
    */
   public void downloadTo(OutputStream outputStream, BlobSourceOption... options) {
-    final CountingOutputStream countingOutputStream = new CountingOutputStream(outputStream);
-    final StorageRpc storageRpc = this.options.getStorageRpcV1();
-    StorageObject pb = getBlobId().toPb();
-    final Map<StorageRpc.Option, ?> requestOptions = StorageImpl.optionMap(getBlobId(), options);
-    ResultRetryAlgorithm<?> algorithm = retryAlgorithmManager.getForObjectsGet(pb, requestOptions);
-    Retrying.run(
-        this.options,
-        algorithm,
-        callable(
-            () -> {
-              storageRpc.read(
-                  pb, requestOptions, countingOutputStream.getCount(), countingOutputStream);
-            }),
-        Function.identity());
+    storage.downloadTo(getBlobId(), outputStream, options);
```

Comment:
After changing the signature on storage, we'll need to convert the options before calling. `com.google.cloud.storage.Blob.BlobSourceOption#toSourceOptions(com.google.cloud.storage.BlobInfo, com.google.cloud.storage.Blob.BlobSourceOption...)` can be used to perform the conversion.